src/formatter/string_formatter.rs

```diff
@@ -1,6 +1,7 @@
+use crate::utils::parallel_map;
 use ansi_term::Style;
+use core::future::Future;
```

Comment:
Should this be `std`?
```suggestion
use std::future::Future;
```